content/actions/reference/events-that-trigger-workflows.md

```diff
@@ -314,6 +314,16 @@ on:
     types: [created, deleted]
 ```
 
+The `issue_comment` event occurs for both, comments on issues as well as pull requests. To run jobs only in one of the cases, you can check the event payload for the `issue.pull_request` attribute and use it as condition to skip jobs.
+
+```yaml
+on: issue_comment
+
+jobs:
+  pr_commented:
```

Comment:
Let's add a comment for readers that skim the workflow file!

```suggestion
  // This job only runs when a pull request comment is created
  pr_commented:
```

Reply:
Think the comments need to be with `#` because it's in a Yaml file

Reply:
`# Comment` here as well

Reply:
Good catch @nschonni @Simran-B 🙈 Yes definitely should be `#`.